src/redisearch_api.c

```diff
@@ -30,7 +32,12 @@ static void valFreeCb(void* unused, void* p) {
 }
 
 static IndexSpec* RS_CreateIndex(const char* name, const RSIndexOptions* options) {
-  RSIndexOptions opts_s = {0};
+  RSIndexOptions opts_s = {
+      .gvcb = NULL,
```

Comment:
you don't need to initialize the other options; just the gcPolicy

Reply:
I think its cleaner this way

Reply:
It's better to just 0-initialize it. This way you know the struct has its defaults; only overrides should be in the initializer.